src/clipboard.js

```diff
@@ -60,12 +60,17 @@ function markAndGetInlineImagesAndRemoveForbiddenOnes($editor, invalidImageSelec
         .find('img[src^="data:image/"]')
         .toArray()
         .map((el) => ({ ...decodeBase64Image(el.getAttribute('src')), el }))
-    images
-        .filter(({ type }) => fileTypes.indexOf(type) === -1 && type !== 'image/svg+xml')
-        .forEach(({ el }) => el.remove())
-    const pngImages = images.filter(({ type }) => fileTypes.indexOf(type) >= 0)
+    images.filter(isForbiddenInlineImage).forEach(({ el }) => el.remove())
+    const pngImages = images.filter(({ type }) => fileTypes.includes(type))
     pngImages.forEach(({ el }) => el.setAttribute('src', loadingImg))
     return pngImages
+
+    function isForbiddenInlineImage({ type, el }) {
```

Comment:
vastauksesta siivottavia kuvia ovat kaikki, jotka ei kuulu fileTypes-listaan paitsi kaavojen svg-kuvat, joita käytetään serverittömässä demossa

Reply:
ehkä tähän voisi laittaa kommentin tai jollain muulla tavoilla ilmoittaa että `el.alt` puuttuminen tarkoittaa kaavojen svg kuvia serverittömässä demossa. Vaikea tulevaisuudessa arvata mitä tuo tarkoittaa. 

Ymmärsinkö oikein edes nyt?

Reply:
toisin päin, eli alt-arvon löytyminen tarkoittaa kaavojen svg-kuvia